docs/resources/confluent_schema_registry_cluster_mode.md

```diff
@@ -79,6 +79,7 @@ The following arguments are supported:
 !> **Warning:** Use Option #2 to avoid exposing sensitive `credentials` value in a state file. When using Option #1, Terraform doesn't encrypt the sensitive `credentials` value of the `confluent_kafka_cluster_mode` resource, so you must keep your state file secure to avoid exposing it. Refer to the [Terraform documentation](https://www.terraform.io/docs/language/state/sensitive-data.html) to learn more about securing your state file.
 
 - `mode` - (Optional String) The global Schema Registry mode. Accepted values are: `READWRITE`, `READONLY`, `READONLY_OVERRIDE`, and `IMPORT`.
+- `force` - (Optional Boolean) An optional flag to force a mode change even if the Schema Registry has existing schemas. This can be useful in disaster recovery (DR) scenarios using [Schema Linking](https://docs.confluent.io/platform/current/schema-registry/schema-linking-cp.html#schema-linking-cp-overview). Defaults to `false`, which does not allow a mode change to `IMPORT` if Schema Registry has registered schemas. Must be unset when importing.
```

Comment:
Do we want to link Cloud docs here instead of Platform?

https://docs.confluent.io/cloud/current/sr/schema-linking.html

Reply:
That's a great idea! I applied the suggestion 👍